private-templates-service/client/src/components/Common/Tags/Tags.tsx

```diff
@@ -45,15 +46,21 @@ interface State {
   newTagName: string;
 }
 
+interface RefsDict {
```

Comment:
Is this interface used anywhere else? If not, I don't think it makes sense to have it - it overcomplicates things imo.